src/sentry/apidocs/parameters.py

```diff
@@ -109,6 +109,44 @@ class OrganizationParams:
 For example the following are valid parameters:
 - `/?project=1234&project=56789`
 - `/?project=-1`
+""",
+    )
+    OWNER = OpenApiParameter(
+        name="owner",
+        location="query",
+        required=False,
+        type=bool,
+        description="""Specify `true` to restricts results to organizations in which you are an owner.""",
+    )
+    QUERY = OpenApiParameter(
+        name="query",
+        location="query",
+        required=False,
+        type=str,
+        description="""The search filter for your query, read more about query syntax [here](https://docs.sentry.io/product/sentry-basics/search/).
+
+Valid query fields include:
+- `id`: The organization ID
+- `slug`: The organization slug
+- `status`: The organization's current status (one of `active`, `pending_deletion`, or `deletion_in_progress`)
+- `email` or `member_id`: Filter your organizations by specific members included
```

Comment:
What ID is `member_id` referring to in this case? Is it the org member id or the user id?

Reply:
org member id, i can specify that (and maybe link the API docs for retrieve an org member since there isn't anywhere in the UI that we surface these)

Reply:
If you query by member_id that would only return a single org right? But if you did it by email, that could return multiple?